server/test/repository/ExportServiceRepositoryTest.java

```diff
@@ -0,0 +1,137 @@
+package repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Locale;
+import java.util.NoSuchElementException;
+import models.LifecycleStage;
+import models.Question;
+import org.junit.Before;
+import org.junit.Test;
+import services.LocalizedStrings;
+import services.question.QuestionOption;
+import services.question.QuestionService;
+import services.question.types.MultiOptionQuestionDefinition;
+import services.question.types.QuestionDefinition;
+import services.question.types.QuestionDefinitionBuilder;
+import services.question.types.QuestionDefinitionConfig;
+
+public class ExportServiceRepositoryTest extends ResetPostgres {
+  private ExportServiceRepository repo;
+  // private Applicant applicant;
+  // private Program program;
+  private QuestionService questionService;
+  private VersionRepository versionRepository;
+
+  @Before
+  public void setUp() {
+    repo = instanceOf(ExportServiceRepository.class);
+    questionService = instanceOf(QuestionService.class);
+    versionRepository = instanceOf(VersionRepository.class);
+  }
+
+  @Test
+  public void getMultiSelectedHeaders_NoPreviousQuestionVersion() {
+    QuestionDefinition questionDefinition =
+        testQuestionBank.applicantKitchenTools().getQuestionDefinition();
+    ImmutableList<String> multiSelectHeaders = repo.getMultiSelectedHeaders(questionDefinition);
+    assertThat(multiSelectHeaders.size()).isEqualTo(3);
+    checkList(multiSelectHeaders, 0, "toaster");
```

Comment:
`containsExactly()` could also do what we're doing with checkList() here. It asserts order.

https://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/ListAssert.html#containsExactly(ELEMENT...)